mypy/nodes.py

```diff
@@ -383,6 +456,29 @@ def accept(self, visitor: NodeVisitor[T]) -> T:
     def is_constructor(self) -> bool:
         return self.info is not None and self._name == '__init__'
 
+    def serialize(self) -> JsonDict:
+        return {'.class': 'FuncDef',
+                'name': self._name,
+                'fullname': self._fullname,
+                'arguments': [a.serialize() for a in self.arguments],
+                'type': None if self.type is None else self.type.serialize(),
+                'is_class': self.is_class,
+                # TODO: Various other flags
```

Comment:
Add other flags, or leave this open? If we leave this open, it might be worth adding an issue for this so we won't forget about it.